common/datablocks/src/data_block.rs

```diff
@@ -120,15 +120,13 @@ impl DataBlock {
     /// Take the first data value of the column.
     pub fn first(&self, col: &str) -> Result<DataValue> {
         let series = self.try_array_by_name(col)?;
```

Comment:
why not use `try_column_by_name`, if the column is Constant, we do not need  to turn it into array.

Reply:
Nice, updated